blueprints/app/files/.eslintrc.js

```diff
@@ -2,13 +2,14 @@
 
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
       legacyDecorators: true,
     },
+    requireConfigFile: false,
```

Comment:
Instead of this, we should try to work towards leveraging the `baber.config.js` support in recent versions of ember-cli-babel.

See https://github.com/rwjblue/___demo-babel-config as a demo of how that can work.

Reply:
Without that change, the configuration for `@babel/eslint-parser` will have to be duplicated like mentioned in https://github.com/babel/ember-cli-babel/issues/366 (specifically https://github.com/babel/ember-cli-babel/issues/366#issuecomment-729839454).

Reply:
Neat! I just copied the changes from https://github.com/rwjblue/___demo-babel-config (following the README) (and also added the `babel.config.js` to `.npmignore`). Note that this is a much larger change now...